README.md

```diff
@@ -0,0 +1,62 @@
+CUPS driver for CAPT printers
+=============================
+
+This is a driver for Canon CAPT-based printers (LBP-****)
+based on several reverse engineering attempts. This is
+currently in an early alpha stage. Use at your own risk.
+
+The printer protocol documentation based on reverse engineering
+is in the SPECS file. This is incomplete and probably has many
+errors. Please help expanding it.
+
+Supported printers
+------------------
+
+  * Canon LBP-2900
+  * Canon LBP-3000
+
+Build
+-----
+
+The project uses GNU Autotools to build the driver.
+Run the following commands after cloning the repo:
+
+```
+$ aclocal
+$ autoconf
+$ automake --add-missing
+```
+
```

Comment:
Maybe add a line about running `./configure` after this?

Reply:
Thanks for reviewing! Honestly, I do not think this step is needed: autoconf configures build parameters and automake builds the driver.

Reply:
Ah bleugh, it's been too long since I've worked with autotools :stuck_out_tongue: 
Right this PR should be fine then.

Reply:
I haven't used autotools in ages, so please forgive me :stuck_out_tongue:

When I run those commands I only end up with a `./configure` and `Makefile.in` files created. To actually compile it I need to run `./configure` then `make` afterwards -- am I doing something wrong?

Reply:
You are right! Thanks for doing a test build. I've added the missing steps to README.md